integration/network/helpers.go

```diff
@@ -9,42 +9,43 @@ import (
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/client"
+	"github.com/docker/docker/testutil"
 	"gotest.tools/v3/assert/cmp"
 	"gotest.tools/v3/icmd"
 )
 
 // CreateMasterDummy creates a dummy network interface
-func CreateMasterDummy(t *testing.T, master string) {
+func CreateMasterDummy(ctx context.Context, t *testing.T, master string) {
 	// ip link add <dummy_name> type dummy
-	icmd.RunCommand("ip", "link", "add", master, "type", "dummy").Assert(t, icmd.Success)
-	icmd.RunCommand("ip", "link", "set", master, "up").Assert(t, icmd.Success)
+	testutil.RunCommand(ctx, "ip", "link", "add", master, "type", "dummy").Assert(t, icmd.Success)
+	testutil.RunCommand(ctx, "ip", "link", "set", master, "up").Assert(t, icmd.Success)
```

Comment:
I see we're replacing `icmd` for our wrapper here, but we don't do that everywhere (e.g., we didn't do it in the `integration-cli`.

- Is that something we want to do across the board? (if so, perhaps we should create a tracking ticket for that)
- Also wondering if passing a context is something that would make sense for `gotest.tools/v3/icmd"` (which we could contribute) 🤔  - probably for a `v4` then 😂